import exceptions

print(exceptions.greet('Beatorice'))